tests/store/artifact/test_artifact_repo.py

```diff
@@ -205,3 +206,30 @@ def list_artifacts(path):
         err_msg = str(exc.value)
         assert _MODEL_FILE in err_msg
         assert _MOCK_ERROR in err_msg
+
+
+@pytest.mark.parametrize("debug", [True, False])
+def test_download_artifacts_provides_traceback_info(debug):
+    if debug:
+        _logger.setLevel(logging.DEBUG)
+    else:
+        _logger.setLevel(logging.INFO)
```

Comment:
QQ, does this ultimately get reset by [this pytest fixture](https://github.com/mlflow/mlflow/blob/f51b8384300fa9378269cf8f63523265e4de6bfa/tests/utils/test_logging_utils.py#L30-L34)? Or does it affect the logging level for future tests - maybe we can try setting the log level to `DEBUG` at the end of a test and making sure it doesn't affect the next ones. I'm pretty sure the fixture should just work due to `autouse=True`, and don't see other tests that call/interact with the fixture, but I also don't see a ton of tests that override the logging level